styleguide/source/_patterns/02-organisms/topic/topic-rail-right/topic-rail-right.twig

```diff
@@ -0,0 +1,3 @@
+<aside class="topic_rail-right">
```

Comment:
Just to clarify, this organism is so that the right rail can be displayed as such in the style guide, but isn't actually used in the template?